### PR TITLE
Added empoche rule

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

**Comment by mimi89999** [comment]
@geek1011 Could you also review this one?

File: jiup/rules/rules.go [hunk 1]
```diff
@@ -457,6 +457,18 @@ func init() {
...
+		v.HTML(
+			"https://empoche.com/download",
+			"#tab-windows",
+			"innerText",
+			h.Re("Empoche\\+Setup\\+([0-9.]+).exe"),
+		),
+		d.Template(
+			"https://empoche-desktop.s3.eu-central-1.amazonaws.com/Empoche+Setup+{{.Version}}.exe",
+			"",
+		),
```
Comment (pgaskin) [inline jiup/rules/rules.go:470]: I'd rather use the HTMLA download extractor here.

Comment (mimi89999) [reply jiup/rules/rules.go:470]: Is it possible to write a nice match for this one?

Comment (pgaskin) [reply jiup/rules/rules.go:470]: `#tab-windows a[href$=".exe"]`. Remember that it uses [Cascadia](https://github.com/andybalholm/cascadia) for matches.

**Pull request merged**